tst/CTA.Rules.Test/PortCoreTests.cs

```diff
@@ -276,6 +275,24 @@ private void TestMvcMusicStore(string version)
             Assert.AreEqual(mvcProjectActions.Count, 4);
         }
 
+        [Test]
+        public void TestAntlrSampleSolution()
+        {
+            var version = "netcoreapp3.1";
```

Comment:
Probably want to follow the example of the other tests and pull out the version to a separate function so we can inject the version number ourselves and run .net 5 if need be.

Reply:
Good point. I will update this.